docs/pages/blog/material-ui-2024-updates.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Material UI 2024 end-of-year updates
+description: Recapping our biggest accomplishments for Material UI this year; plus, a look at what's to come in 2025.
+date: 2024-12-11T00:00:00.000Z
+authors: ['diegoandai']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+As the end of the year approaches, we want to take some time to review some of this year's highlights for Material UI and update you on what you can expect moving forward.
+
+## Material UI v6 release and support
+
+We released Material UI v6 in August, offering new features, fixes, and a smoother migration experience. 
+So far, almost a million users have successfully migrated.
+
+The reception for the new version has been very positive, and we hope many more users can migrate in the upcoming months.
+
+Our focus has been on responding to v6-related issues to support your migration. We thank everyone in the community who has helped by reporting or fixing problems and encourage everyone else to migrate. We are happy and ready to help!
+
+## React 19 support
+
+As of today, Material UI v5 and v6 support React 19. This has been one of our main focuses during the second half of 2024.
+
+We've worked hard to ensure that the upgrade experience is as seamless as possible, and we're very grateful to all the community members who've been testing it out and reporting issues.
+
+## Material Design 3
+
+A year ago, we shared that we expected to have a Material Design 3 implementation ready by the end of 2024. 
+Due to the more pressing efforts mentioned above, we have not been able to deliver on this yet.
+
+Updating the library's look and feel in such a big way is an incredibly complex undertaking that can't be rushed.
+We're committed to providing the best experience for all users and use cases, and we're working hard to determine the best way to accomplish that.
+This is one of our top priorities for 2025, and we'll keep you updated on our progress.
```

Comment:
One final question for @aarongarciah: This section is ambiguous (by design), as we don't want to make false promises. What do you think about shortly acknowledging this? For example:

```suggestion
Updating the library's look and feel in such a big way is an incredibly complex undertaking that can't be rushed.
We're committed to providing the best experience for all users and use cases, and we're working hard to determine the best way to accomplish that.

We know this provides little detail into what to expect.
That's because we are still iterating on the solution, and even though we have an idea of the direction, we want to be sure of it before sharing more.
This is one of our top priorities for 2025, and we'll keep you updated on our progress.
We hope you understand.
```

This sounds a bit more apologetic, but I think it's the right tone to address the users that have been waiting for this for a long time.

Reply:
For me as a user/reader, the phrasing above would make me lose some confidence in the project. How about a compromise somewhere in the middle that acknowledges the sparse details and kindly requests more patience? 😄

```suggestion
Updating the library's look and feel in such a big way is an incredibly complex undertaking that can't be rushed.
We're committed to providing the best experience for all users and use cases, and we're working hard to determine the best way to accomplish that.

This is one of our top priorities for 2025, and we'll keep you updated on our progress as soon as we have more definitive details to share.
Thank you for your continued patience.
```

Reply:
Sam's version works for me 🆗